src/Offer/ReadModel/JSONLD/OfferLDProjector.php

```diff
@@ -631,7 +631,11 @@ protected function applyBookingInfoUpdated(AbstractBookingInfoUpdated $bookingIn
 
         $offerLd = $document->getBody();
 
-        $offerLd->bookingInfo = $bookingInfoUpdated->getBookingInfo()->toJsonLd();
+        if (empty($bookingInfoUpdated->getBookingInfo()->toJsonLd())) {
```

Comment:
I think this is something that can also be tested with a unit test on `OfferLDProjectorTest`?

Reply:
Not only did I provide a unit test for this, I also added another unit test to add bookinginfo, because this seemed to be missing.